pkg/channel/consolidated/reconciler/controller/kafkachannel.go

```diff
@@ -595,3 +617,133 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, kc *v1beta1.KafkaChannel)
 	}
 	return newReconciledNormal(kc.Namespace, kc.Name) //ok to remove finalizer
 }
+
+func (r *Reconciler) updateMonitoringStatus(ctx context.Context, configMap *corev1.ConfigMap) {
+	backend := configMap.Data["metrics.backend-destination"]
+	if backend == "none" || backend == "opencensus" {
```

Comment:
I know it is probably not only the Kafka channel, but why do we do nothing if the backend is opencensus?

Reply:
We dont support opecensus backed downstream.